tests/faucet_mininet_test.py

```diff
@@ -185,22 +188,21 @@ def one_ipv6_ping(self, host, dst):
     def one_ipv6_controller_ping(self, host):
         self.one_ipv6_ping(host, self.CONTROLLER_IPV6)
 
-    def wait_until_matching_flow(self, flow, timeout=5):
+    def wait_until_matching_flow(self, exp_flow, timeout=5):
         # TODO: actually verify flows were communicated to the physical switch.
```

Comment:
Please remove this TODO and the if SWITCH_MAP: here, because your very nice change fixes the problem for hardware also.